Ah, booh

```
cmd/dockerd/daemon_unix.go:160:2: printf: github.com/docker/docker/vendor/github.com/sirupsen/logrus.Info call has possible formatting directive %s (govet)
	logrus.Info("started managed containerd, listening on %s", cli.ContainerdAddr)
	^
```

--- a/libcontainerd/supervisor/remote_daemon.go
+++ b/libcontainerd/supervisor/remote_daemon.go
@@ -39,7 +39,6 @@ type remote struct {
 	daemonStartCh chan error
 	daemonStopCh  chan struct{}
 
-	rootDir  string
 	stateDir string
 }
 
@@ -55,7 +54,6 @@ type DaemonOpt func(c *remote) error
 // Start starts a containerd daemon and monitors it
 func Start(ctx context.Context, rootDir, stateDir string, opts ...DaemonOpt) (Daemon, error) {
 	r := &remote{
-		rootDir:  rootDir,
 		stateDir: stateDir,
 		Config: config.Config{
 			Version: 2,

--- a/libcontainerd/supervisor/remote_daemon.go
+++ b/libcontainerd/supervisor/remote_daemon.go
@@ -9,7 +9,6 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
-	"sync"
 	"time"
 
 	"github.com/containerd/containerd"
@@ -31,7 +30,6 @@ const (
 )
 
 type remote struct {
-	sync.RWMutex
 	config.Config
 
 	daemonPid int

--- a/libcontainerd/supervisor/remote_daemon_linux.go
+++ b/libcontainerd/supervisor/remote_daemon_linux.go
@@ -55,5 +55,5 @@ func (r *remote) killDaemon() {
 }
 
 func (r *remote) platformCleanup() {
-	os.Remove(filepath.Join(r.stateDir, sockFile))
+	_ = os.Remove(r.Address())
 }

